apps/tailwind-components/components/input/CheckboxIcon.vue

```diff
@@ -50,5 +56,6 @@ defineProps<{
   invalid?: boolean;
   valid?: boolean;
   disabled?: boolean;
+  focused?: boolean;
```

Comment:
prop appears unused

Reply:
Good catch. This was from an earlier experiment. I solved this using a different approach, but forgot to remove the prop.